test/Sema/subscripting.swift

```diff
@@ -0,0 +1,16 @@
+// RUN: %target-run-simple-swift | %FileCheck %s
```

Comment:
Nitpick: test/Sema isn't supposed to have execution tests. Please move this to test/Interpreter (which has a number of non-interpreter execution tests at the moment) and add "REQUIRES: executable_test".